fluentd/configs.d/openshift/output-applications.conf

```diff
@@ -0,0 +1,5 @@
+<match **>
+   @type copy
+   @include output-es-config.conf
+   @include ../dynamic/es-copy-config.conf
```

Comment:
True, but

- the same thing applies to any user provided configuration, including the support this commit is providing - the user can't use any plugin they want in their custom input-post-foo.conf, they have to use the ones we provide
- there are some things they can do with the plugins built-in to fluentd: out_stream, out_roundrobin, out_file, out_exec, out_forward, in addition to es.

If we give users the ability to provide their own config, we should also give them a way to provide their own plugins . . . http://docs.fluentd.org/articles/plugin-management